indexer/feature_utils.cpp

```diff
@@ -438,4 +459,18 @@ string GetLocalizedFeeType(TypesHolder const & types)
   return localized_types[0];
 }
 
+string GetLocalizedSecondaryAtm(TypesHolder const & types)
+{
+  auto const & isAtmType = ftypes::IsATMChecker::Instance();
+  if (HasType(isAtmType, types) && !HasMainType(isAtmType, types))
+    return GetLocalizedString("type.amenity.atm");
+  else
+    return "";
+}
+
+string GetLocalizedString(std::string const & key)
+{
+    return platform::GetLocalizedString(key);
+}
```

Comment:
I doubt that we need additional wrapper function around platform::GetLocalizedString

Reply:
Done